### ISSUE TITLE
Missing Code for Task Description Generation in digirl/environment/android/assets/task_set Folder

### ISSUE THREAD

**Issue by Youngjae-KAIST** [DigiRL-agent/digirl#31]
I noticed that the code for generating task descriptions in the assets/task_set folder is not included in the repository. In the paper, it is mentioned that three basic operations were used to "post-process" the tasks in the AITW dataset. However, I couldn't find any code that reflects this post-processing in the current codebase.

The task descriptions seem to have been created by enumerating basic operations and replacing placeholders with possible values. This raises a couple of questions:

Is there a one-to-one correspondence between the provided dataset and the original AITW webshopping dataset? If not, it might be helpful to clarify this in the documentation.

Could you provide the code or scripts used for generating the task descriptions? Having access to this would be beneficial for understanding and reproducing the results presented in the paper.

If the original AITW dataset cannot be clearly identified from this dataset or if the provided dataset differs significantly, perhaps it would be appropriate to reconsider naming it the AITW dataset to prevent any potential confusion.

**Comment by BiEchi** [comment]
Thanks for your interest in our work. The AitW General task set is kept the same as the original task set. The AitW Webshop task is post-processed. The post-processing includes (i) filtering out websites that has a high potential of blocking digital agent requests, (ii) identifying what operations are actually doable by an agent, which in the end we found 3 operations for each website, and (ii) adding curriculum upon these tasks (from an easy task to a hard task). We did not reserve the task set generation script.
